structural-variantcalling.wdl

```diff
@@ -0,0 +1,101 @@
+version 1.0
+
+import "tasks/common.wdl" as common
+import "tasks/delly.wdl" as delly
+import "tasks/manta.wdl" as manta
+import "tasks/picard.wdl" as picard
+import "tasks/bcftools.wdl" as bcftools
+import "tasks/survivor.wdl" as survivor
+import "tasks/clever.wdl" as clever
+import "tasks/bwa.wdl" as bwa
+import "tasks/samtools.wdl" as samtools
+
+workflow SVcalling {
+    input {
+        File bamFile
+        File bamIndex
+        File referenceFasta
+        File referenceFastaFai
+        File referenceFastaDict
+        BwaIndex bwaIndex
+        String sample
+        String outputDir = "."
+    }
+     
+    call delly.CallSV as delly {
+        input:
+            bamFile = bamFile,
+            bamIndex = bamIndex,
+            referenceFasta = referenceFasta,
+            referenceFastaFai = referenceFastaFai,
+            outputPath = outputDir + 'structural-variants/delly/' + sample + ".delly.bcf"
```

Comment:
There should be a `/` before `structural-variants`, otherwise the output will end up in a hidden folder